migration/makefile

```diff
@@ -0,0 +1,6 @@
+test:
```

Comment:
The toplevel make should invoke this test so that the CI build will run these unit-tests